sycl/include/CL/sycl/detail/pi.h

```diff
@@ -86,41 +86,10 @@ using pi_native_handle = uintptr_t;
 // TODO: populate PI enums.
 //
```

Comment:
Is it still relevant?

Reply:
I don't think it is, so I have removed it. Any missing ones should be added by need.